.github/workflows/ci.yml

```diff
@@ -21,7 +21,7 @@ jobs:
         # deps: [ highest ]
         # without-dama: [ 0 ]
         # use-phpunit-extension: [ 0 ]
-        # phpunit: [ 9 ]
+        # phpunit: [ 11 ]
```

Comment:
I think that from now, PHPUnit 11 should be the default phpunit version on which we're running tests